### ISSUE TITLE
toJsonString ignored fieldConverter?

### ISSUE THREAD

**Issue by nielinjie** [cbeust/klaxon#221]
I am trying to follow the example in README. However got the date field does not converted.
`{"birth" : "Fri Nov 30 11:50:42 CST 2018", "name" : "hha"}`

Here is my code, do I miss something?

```
    import com.beust.klaxon.Converter
    import com.beust.klaxon.JsonValue
    import com.beust.klaxon.Klaxon
    import com.beust.klaxon.KlaxonException
    import java.text.SimpleDateFormat
    import java.util.*

    class WithDate @JvmOverloads constructor(val name: String, @ESDate val birth: Date)


    val dateConverter = object : Converter {
        val format = SimpleDateFormat("yyyy/MM/dd")

        override fun canConvert(cls: Class<*>): Boolean {
            return cls == Date::class.java
        }

        override fun fromJson(jv: JsonValue) =
            throw KlaxonException("Couldn't parse date: ${jv.string}")

        override fun toJson(o: Any): String {
            return format.format(o as Date)
        }
    }

    fun main(args: Array<String>) {
        val k = Klaxon()
            .fieldConverter(ESDate::class, dateConverter)
        println(k.toJsonString(WithDate("hha", Date())))
    }


    @Target(AnnotationTarget.FIELD)
    annotation class ESDate
```

**Comment by nielinjie** [comment]
I have followed into code at line 223 in Klaxon.kt. I see no property pass into `findConverter` method. 
```
fun toJsonString(value: Any): String = toJsonString(value, findConverter(value))
```

**Comment by cbeust** [comment]
Fixed in 4.0.2.
